src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -475,52 +477,52 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		push(exposureSubmissionHotlineViewController)
 	}
 	
-    private func makeSRSConsentScreen(srsFlowType: SRSFlowType) -> UIViewController {
-        let srsConsentViewController = SRSConsentViewController(
-            onPrimaryButtonTap: { [weak self] isLoading in
-                guard let self = self else { return }
-                
-                isLoading(true)
-                self.model.exposureSubmissionService.getTemporaryExposureKeys { error in
-                    isLoading(false)
-                    
-                    guard let error = error else {
-                        switch srsFlowType {
-                        case .srsPositive:
-                            self.model.storeSelectedSRSSubmissionType(.srsSelfTest)
-                            self.showSRSFlowNextScreen()
-                        case .positiveWithoutResultInTheApp:
-                            self.showSRSTestTypeSelectionScreen()
-                        }
-                        return
-                    }
-                    
-                    // User selected "Don't Share" / "Nicht teilen"
-                    if error == .notAuthorized {
-                        Log.info("OS submission authorization was declined.")
-                    } else {
-                        self.showErrorAlert(for: error)
-                    }
-                }
-            },
-            dismiss: { [weak self] in self?.dismiss() }
-        )
-        
-        let footerViewModel = FooterViewModel(
-            primaryButtonName: AppStrings.ExposureSubmissionTestResultAvailable.primaryButtonTitle,
-            primaryIdentifier: AccessibilityIdentifiers.ExposureSubmissionTestResultAvailable.primaryButton,
-            isSecondaryButtonEnabled: false,
-            isSecondaryButtonHidden: true
-        )
-        
-        let topBottomContainerViewController = TopBottomContainerViewController(
-            topController: srsConsentViewController,
-            bottomController: FooterViewController(footerViewModel)
-        )
-        
-        return topBottomContainerViewController
-    }
-    
+	private func makeSRSConsentScreen(srsFlowType: SRSFlowType) -> UIViewController {
+		let srsConsentViewController = SRSConsentViewController(
+			onPrimaryButtonTap: { [weak self] isLoading in
+				guard let self = self else { return }
+				
+				isLoading(true)
+				self.model.exposureSubmissionService.getTemporaryExposureKeys { error in
+					isLoading(false)
+					
+					guard let error = error else {
+						switch srsFlowType {
+						case .srsPositive:
+							self.model.storeSelectedSRSSubmissionType(.srsSelfTest)
+							self.checkSRSPrerequisitesToContinueSRSFlow(isSelfTestTypePreselected: true, isLoading: isLoading)
```

Comment:
We do we need to set it true? we don't show the type selection screen. Better to set the default to false and just pass the isLoading here

Reply:
You're right, that was still the old flow with the self test preselection.

Reply:
Fixed